pkg/datapath/linux/node.go

```diff
@@ -946,9 +951,246 @@ func (n *linuxNodeHandler) nodeUpdate(oldNode, newNode *nodeTypes.Node, firstAdd
 		}
 	}
 
+	if option.Config.IPAM == ipamOption.IPAMENI {
+		addedInterfaces, removedInterfaces := diffInterfaces(oldNode.Interfaces, newNode.Interfaces)
+
+		// Configure new interfaces.
+		for _, addedInterface := range addedInterfaces {
+			mtu := 1500 // FIXME where do we get the MTU from?
+			if _, err := linuxrouting.RetrieveIfIndexFromMAC(addedInterface.MAC, mtu); err != nil {
+				return err
```

Comment:
We should not error out but just warn. We shouldn't skip the node handler because we already have the ifindex so we can continue.